cpp/src/barretenberg/ecc/fields/field_impl.hpp

```diff
@@ -607,4 +606,29 @@ template <class T> constexpr field<T> field<T>::multiplicative_generator() noexc
     return target;
 }
 
+// For serialization
+template <class Params> void field<Params>::msgpack_pack(auto& packer) const
+{
+    auto adjusted = from_montgomery_form();
+    uint64_t bin_data[4] = {
+        htonll(adjusted.data[3]), htonll(adjusted.data[2]), htonll(adjusted.data[1]), htonll(adjusted.data[0])
```

Comment:
This is required by barretenberg :) I made sure the toBuffer and fromBuffer formats were exactly the same. 'network byte order' just means big endian in practicality, it's how bberg already does it in serialize.hpp. But yeah overall just replicating bberg format exactly because the typescript code still supports both buffer formats, so made sense to align them

Reply:
Alright, thanks for explaining.